mrp_project/wizard/mrp_production_createproject.py

```diff
@@ -0,0 +1,73 @@
+from odoo import api, fields, models, _
```

Comment:
Thank you for your contribution
please consider to add the License for this file
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html

Reply:
One moment.

Reply:
Donde.